test/load/terraform/test-servers.tf

```diff
@@ -37,13 +37,13 @@ resource "aws_security_group" "test-servers" {
     from_port   = 22
     to_port     = 22
     protocol    = "6"
-    cidr_blocks = [var.vpc_cidr]
+    cidr_blocks = ["0.0.0.0/0"]
   }
   egress {
     from_port   = 0
     to_port     = 0
     protocol    = "-1"
-    cidr_blocks = [var.vpc_cidr]
+    cidr_blocks = ["0.0.0.0/0"]
```

Comment:
I'm wondering why this was set to `var.vpc_cidr` in the first place? Shouldn't this be more restrictive than allowing "anywhere"?

Reply:
It was set to vpc_cidr (="10.0.0.1/16") because of the security alert. However, we have to allow ingress on port 22 in order to allow the terraform provisioner to run the workload command https://github.com/hashicorp/consul/blob/915f1089f03f25b3ca0bf6edd7c719d0c9b00f4e/test/load/terraform/test-servers.tf#L57-L69

The egress is to allow the test server to run some install script: https://github.com/hashicorp/consul/blob/main/test/load/packer/loadtest-ami/scripts/install-k6.sh 

I think the consul cluster instances allows this too for downloading consul: https://github.com/hashicorp/terraform-aws-consul/blob/81b6173e8f0869ce96cbc93545dce4b8f68a16e0/modules/consul-cluster/main.tf#L173-L178

Reply:
Sorry, I probably should have put this comment on the line for the ingress block. I was hoping we could restrict the ingress to something other than "anywhere". For now, if we kept it as a variable instead of hard-coding it as "0.0.0.0" we could let the user set it in the `var.tfvars` file. What do you think about this option?

Reply:
@eculver , agreed. So I added a new variable named `vpc_allwed_ssh_cidr` and set it to `10.0.0.0/16` by default and add some note in `test/load/terraform/README.md` (please see the last commit for the change)